lib/etl/page_content_extractor.rb

```diff
@@ -0,0 +1,21 @@
+class PageContentExtractor
+  def initialize(parsed_book)
+    @book_text = ''
+    @parsed_book = parsed_book
+  end
+
+  def start
+    parsed_book.each_page_on_spine do |page|
+      print '.'
+      parsed_xml = page.content_document.read
+      page_plain = ActionView::Base.full_sanitizer.sanitize(parsed_xml)
```

Comment:
We're sanitizing/removing tags here because of two reasons:

1. We don't want ElasticSearch to be searching on XML/HTML tags
2. If we're displaying context around our search results, we don't want to display XML/HTML tags, but rather just the content itself.